src/App.js

```diff
@@ -35,12 +35,23 @@ import LogoutPage from './pages/Logout/Logout'
 import { mapSizesToProps } from './utils/withSizes'
 import styles from './App.module.scss'
 import './App.scss'
-
-const PATHS = {
-  FEED: '/feed'
+import CreateAccountFreeTrial from './pages/Login/CreateAccountFreeTrial'
+
+export const PATHS = {
+  FEED: '/feed',
+  LOGIN: '/login',
+  LOGIN_VIA_EMAIL: '/login/email',
+  CREATE_ACCOUNT_FREE_TRIAL: '/signup',
```

Comment:
```suggestion
  CREATE_ACCOUNT_FREE_TRIAL: '/sign-up',
```

Reply:
We can use `CREATE_ACCOUNT` name, more general (if we decide delete trial feature in  future)